src/docker-compose-cli.js

```diff
@@ -1,6 +1,6 @@
 const childProcess = require('child_process');
 
-const DOCKER_COMPOSE_CLI = 'docker-compose';
+const DOCKER_CLI = 'docker';
```

Comment:
Is there any sense to declare this here and not declare the `compose` on line 13?